packages/mobile/src/screens/chat-screen/ReactionPopup.tsx

```diff
@@ -95,24 +99,47 @@ export const ReactionPopup = ({
     (r) => r.user_id === userIdEncoded
   )?.reaction
 
-  const beginAnimation = useCallback(() => {
-    Animated.spring(backgroundOpacityAnim.current, {
+  const beginMountAnimation = useCallback(() => {
+    Animated.timing(backgroundOpacityAnim.current, {
       toValue: BACKGROUND_OPACITY,
-      useNativeDriver: true
+      useNativeDriver: true,
+      duration: MOUNT_ANIMATION_DURATION_MS
     }).start()
-    Animated.spring(otherOpacity.current, {
+    Animated.timing(otherOpacity.current, {
       toValue: 1,
-      useNativeDriver: true
+      useNativeDriver: true,
+      duration: MOUNT_ANIMATION_DURATION_MS
     }).start()
     Animated.spring(translationAnim.current, {
       toValue: 0,
       useNativeDriver: true
     }).start()
   }, [])
 
+  const beginUnmountAnimation = useCallback(() => {
+    Animated.timing(backgroundOpacityAnim.current, {
+      toValue: 0,
+      useNativeDriver: true,
+      duration: UNMOUNT_ANIMATION_DURATION_MS
+    }).start()
+    Animated.timing(otherOpacity.current, {
+      toValue: 0,
+      useNativeDriver: true,
+      duration: UNMOUNT_ANIMATION_DURATION_MS
+    }).start(onClose)
+    Animated.spring(translationAnim.current, {
+      toValue: REACTION_CONTAINER_HEIGHT,
+      useNativeDriver: true
+    }).start()
```

Comment:
And same thing here, can have the callback on the start for the .parallel part

Reply:
Had to keep the translation animation separate bc it takes long to complete, and while it's incomplete the user wouldn't be able to open a new reaction popup. Couldn't find a way to cut off the animation time or change the duration (not sure we would want to do that anyway) so this seemed like the best solution.